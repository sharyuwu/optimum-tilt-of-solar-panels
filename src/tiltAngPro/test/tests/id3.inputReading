id3_1
2020
02
28
2021
02
28
id3_2
1996
01
03
2000
01
14