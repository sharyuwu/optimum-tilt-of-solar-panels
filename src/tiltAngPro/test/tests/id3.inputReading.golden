.inputReading
id3_1
Input 2020-02-28 Absolute Erros = 0
Input 2021-02-28 Absolute Erros = 0
id3_2
Input 1996-01-03 Absolute Erros = 0
Input 2000-01-14 Absolute Erros = 0
